notebooks/200_models/201_fastflow.ipynb

```diff
@@ -5,18 +5,14 @@
    "cell_type": "markdown",
```

Comment:
<p>This still mentions the old visualizer callback</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/1043/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>